sigs.yaml

```diff
@@ -1080,6 +1080,15 @@ sigs:
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-gcp/master/OWNERS
+    meetings:
+    - description: Cluster API Provider GCP office hours
+      day: Thursday
+      time: "10:00"
+      tz: PT (Pacific Time)
+      frequency: monthly on 1st Thursday
```

Comment:
Hm, you had it on Wed earlier. See here:
https://github.com/kubernetes/community/pull/6582#pullrequestreview-929071725
Its scheduled for today.

What day is preferable?

Reply:
yes, was my mistake, it should be in the first Thursday of the month